src/components/TextInput/TextInput.stories.mdx

```diff
@@ -404,13 +405,44 @@ Use the `className` prop to add a custom class, or classes to an input.
       const [value1, setValue1] = useState('');
       const [value2, setValue2] = useState('');
       const [value3, setValue3] = useState('');
+      return (
+        <Box
+          display="flex"
+          direction="row"
+          justify="between"
+          overflow="hidden"
+          childGap="md"
+        >
+          <TextInput
+            value={value1}
+            label="input 1"
+            onChange={event => setValue1(event.target.value)}
+          />
+          <TextInput
+            value={value2}
+            label="input 2"
+            onChange={event => setValue2(event.target.value)}
+          />
+        </Box>
+      );
+    }}
+  </Story>
+</Canvas>
+
+## Sizes
+
+Set the size of the text input to `sm`, `md` or `lg`. `md` is the default size.
```

Comment:
Since there is a "pattern" story with TextInputs and Buttons together, the `Buttons` should not be included.  This set of stories should be TextInput specific.

Reply:
removed these in my commit.